b2/console_tool.py

```diff
@@ -1119,7 +1178,7 @@ class Sync(Command):
     is a regular expression that is tested against the full path
     of each file.
 
-    Note that --includeRegex cannot be used without --excludeRegex.
+    Note that --includeRegex is skipped without --excludeRegex.
```

Comment:
it has to be an error

Reply:
As discussed with @ppolewicz , I'm going to implement this assertion in SDK, not CLI.

Reply:
I've reverted this line.